packages/frontend-v2/components/ScheduleCourse/ScheduleCourse.tsx

```diff
@@ -57,6 +55,7 @@ interface ScheduleCourseProps extends DraggableScheduleCourseProps {
   attributes?: any;
   transform?: string;
   isDisabled: boolean;
+  isOverlay?: boolean;
```

Comment:
I had to add a type to the props here because for whatever reason isDragging from the useDraggable hook is briefly the opposite of what it should be when the drag starts, causing some visual jank.

Reply:
lmfao what even